kong/tracing/instrumentation.lua

```diff
@@ -82,6 +83,15 @@ function _M.balancer(ctx)
   local try_count = balancer_data.try_count
   local upstream_connect_time = split(var.upstream_connect_time, ", ", "jo")
 
+  local last_try_balancer_span
```

Comment:
Nice to have those decoupled from the OTEL plugin. Eventually, we will also decouple the creation of root span and "last_try_balancer_span` too.